content/news/2020-09-28-wallpapers-contest.md

```diff
@@ -0,0 +1,34 @@
++++
+type = "news"
+title = "Call for Wallpapers"
+author = "bitigchi"
+date = "2020-10-01 13:00:00+03:00"
+tags = ["wallpapers", "contest"]
++++
+
+One of the most common feedback that we receive is that the Haiku needs some nice colourful wallpapers shipping with Haiku releases. Suffice to say, that we agree with this completely. We definitely want to improve the current situation. And we need your help.
+
+Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned on the screen on an appropriate position according to the screen resolution. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
+
+We are looking for artists that can contribute to Haiku with some appealing wallpapers. To be more precise, we are looking for either:
+
+- Vector illustrations
+- Painted illustrations
+
+## Guidelines
+
+Artists are free to submit in any category they please. We hope below guidelines will help you in creating the new Haiku wallpapers:
+
+- Illustrations should be in at least landscape 5K size.
+- Vector illustrations should include a "lite" version, to be used as the default wallpaper (see the second paragraph). Color scheme of this lite variant should blend in with the standard system background colour.
```

Comment:
```suggestion
- Vector illustrations should include a "lite" version, to be used as the default wallpaper (see the second paragraph). The colour scheme of this lite variant should blend in with the standard system background colour.
```

Maybe quickly summarize what's in the 2nd paragraph. Do you mean "(as a logo placed on the default background)" ?